### ISSUE TITLE
Customization of threads as tree-branches for easier overview of whole conversations

### ISSUE THREAD

**Issue by panomitrius** [RocketChat/feature-requests#429]
**Is your feature request related to a problem? Please describe.**
Short version: Group conversations are hard to navigate in the current thread-structure.
More details: I’m experiencing with a Rocket Chat server (using Ubuntu and Docker) for the purpose of having a communication platform for philosophical-existential questions and many of the comments in the thread take on new meaning and it would be great if there was some way I could set it up to be more like branches. At the moment it’s quite hard to follow what’s going on in the conversation.

**Describe the solution you'd like**
I'd like Rocket Chat’s thread system to be able to be configured so it becomes more like a tree-branch structure, similar to that of reddit. 

**Describe alternatives you've considered**
Maybe this could be in the form of an addon/plugin?

**Comment by crysis-ps** [comment]
Would be very useful
